src/database/DataTypes/Transaction.js

```diff
@@ -514,6 +514,7 @@ Transaction.schema = {
     insuranceDiscountRate: { type: 'double', optional: true },
     insuranceDiscountAmount: { type: 'double', optional: true },
     paymentType: { type: 'PaymentType', optional: true },
+    isCancellation: { type: 'bool', default: false },
```

Comment:
The only difference would be for calculating a patients credit- everything else is summed and so would be fine, I believe

Except I just realised that if mobile were to really get a cancellation with negative values there might be some more serious consequences since all number parsing pretty much does positive integers.. something to look into buy definitely out of scope.